### ISSUE TITLE
Need Asset Type for Air Purifiers 

### ISSUE THREAD

**Issue by jartam** [theodi/BDNS#88]
Hi there,

We purchased portable "Air Purifiers" that aren't documented in the asset type v2.3 standard for the Bay Area to purify and provide additional ventilation in R&D/Lab spaces primarily here in the Bay Area (Here is the link of the product: https://www.blueair.com/us/air-purifiers/classic-680i/1886.html). We would like to use "AP" as the asset type abbreviations so we can provide QR codes accordingly. Typically though for the BOS naming abbreviation you want a property associated with asset. These are portable and can be moved at anytime. So I have a couple of questions surrounding this request:

1) Can we use "AP" as the asset type?
2) Here is the BOS naming standard v1.3 - below. For "site" since these are portable: can we enter portable for site location? These will be moved from location to location. We would track actual location on a separate Google sheet if that is acceptable. 

"asset": {
    "guid": "ifc://04aEp5ymD_$u5IxhJN2aGi",
    "site": "UK-LON-BNG1",
    "name": "AHU-1"

Also it's ok if Google decides not to want these tracked in LBAM since they are portable and not "fixed" assets. We can just track them for internal purposes only here in the Bay Area if warranted. Please let me know. Thanks. 

-Justin 
Cell 650-930-6893

**Comment by RitaLav** [comment]
Point 1: 'AP' is currently reserved for 'Alarm Panel'.

Ideas for abbreviation/discussion: 'AIRP', or 'APUR'

**Comment by jartam** [comment]
AIRP would work fine too. What about the asset location issue? Does Google want these portable air purifiers tracked in LBAM?

**Comment by RitaLav** [comment]
Point 2 refers to a different standard. @richardkreid @pisuke could you advise which is the correct forum to raise item 2?

**Comment by pisuke** [comment]
Hi @jartam are these portable across buildings/sites or do they move across sites? Do they move from city to city or across countries?
If they stay in a building/site, that building could be added. Otherwise maybe we should just use something like UK-LON-NOSITE

**Comment by jartam** [comment]
Yes Francesco - these are portable and not fixed - these air purifiers
would move across from site to site.

Just need a process for "No Site" and what abbreviation to use for these
air purifiers: "airp" as an example.

-Justin

**Comment by RitaLav** [comment]
AIRP (air purifier) will be added to the abbreviation register as per discussion above.

**Comment by jgunstone** [comment]
just had a look at adding this - 

convinced myself that the correct Ifc type would be:
[IfcFilterType/AIRPARTICLEFILTER](https://standards.buildingsmart.org/IFC/RELEASE/IFC4_1/FINAL/HTML/schema/ifchvacdomain/lexical/ifcfiltertype.htm)

given that an air-purifier just actively circulates air through a filter.... (though the active bit isn't very well captured here...) 
then realised: 
filter - air purifier unit, APU,,IfcFilter,NOTDEFINED

is already in BDNS - @RitaLav - do you think that would do?

**Comment by RitaLav** [comment]
You are correct, it would be a duplicate - "APU" can be used for air purifier.
